src/sas/qtgui/Perspectives/Fitting/SmearingWidget.py

```diff
@@ -321,6 +314,23 @@ def setState(self, smearing, accuracy, d_down, d_up):
         if d_up is not None:
             self.model.item(MODEL.index('PINHOLE_MAX')).setText(d_up)
 
+    def onDQSmear(self):
+        """
+        Create a custom dQ smear object that will change the way residuals
+        are compute when fitting
+        """
+        # resolution information already in data.dx (if 1D) or 
+        # data.dqx_data & data.dqy_data (if 2D),
+        # so only need to set accuracy for 2D
+        _, accuracy, _, _ = self.state()
+        self.current_smearer = smear_selection(self.data, self.kernel_model)
+        if isinstance(self.data, Data2D):
+            backend_accuracy = ACCURACY_DICT.get(accuracy)
+            if backend_accuracy:
+                self.current_smearer.set_accuracy(accuracy=backend_accuracy)
+            else:
+                self.current_smearer.set_accuracy(accuracy='low')
```

Comment:
Why is this necessary? ACCURACY_DICT contains all possible keys, and in case we don't find the right key, I think it is better to just stay with the previously defined accuracy level, rather than reset it quietly?
(Also, below similar suggestion)

Reply:
See above.